project/build.scala

```diff
@@ -32,7 +32,7 @@ object build extends Build {
     tokens,
     transversers,
     trees
-  )
+  ) dependsOn (scalameta)
```

Comment:
Because, when I do `sbt console`, I can't directly do things like `import scala.meta._`. I'd have to do `scalameta/console` without this.